### ISSUE TITLE
wbtc stuck on worm hole bridge

### ISSUE THREAD

**Issue by mouzergrodno** [solana-labs/oyster#347]
Hi! On september 15th I started a transfer of 0.6 wbtc from my ethereum address on metamask to my solana address on phantom wallet.....transaction was stuck for a few hours....it was saying that it's waiting for 15th confirmation....and then like 6-8 hrs later I refreshed the page and there was no history of this transaction, and on etherscan it shows that wbtc was transferred from ethereum to wormhole bridge, but didn't show up on my solana address....
Here's my transaction hash on ethereum blockchain
0x9212ad71d5c4949b1c4e2c92162b78b5eb4103ce1eded7d6b876930c9dfe9eb1

my solana address is - 9fw3rTt7uivxQFUK315KV7EeU2Xd2qL7JRF2oohMYTPe
on solana explorer I don't see a wrapped btc token on my address as well
What should I do?

**Comment by mouzergrodno** [comment]
there's no transaction signature for this transaction on solana explorer, checked all the transactions for that date...so it was confirmed on ethereum blockchain but wasn't confirmed on solana I believe.......waiting for your help...thank you!

**Comment by leoluk** [comment]
We're still looking into this.

**Comment by mouzergrodno** [comment]
Ok...thanks for a quick answer, really appreciate....keep me updated!

**Comment by sunmilee** [comment]
Has this been resolved? I have the same issue https://github.com/solana-labs/oyster/issues/325 and I haven't gotten an update in a while

**Comment by mouzergrodno** [comment]
No, it wasn't! As I understand - we need to wait for a software upgrade on v1. Let's ask Leoluk what's approximate ETA on this??

**Comment by mouzergrodno** [comment]
@leoluk 
Hi! It's been almost half a year since my wbtc got stuck in the bridge. Do you have an ETA when you gonna have an upgrade and when I gonna get my funds back? So I could plan my budget....is it gonna happen this coming spring or later?.....any updates on this would be much appreciated!

**Comment by leoluk** [comment]
Next couple weeks. Apologies for leaving you hanging for so long!

**Comment by leoluk** [comment]
We just implemented the necessary code for v2 and can easily back port it to v1 to get this sorted :)

**Comment by mouzergrodno** [comment]
Ok...thank you! 
So when it's all gonna be done, I will be able to redeem my tokens on v2bridge page myself right?

**Comment by leoluk** [comment]
The transfer will complete on v1 and then you can either send them back or migrate them to v2 tokens

**Comment by mouzergrodno** [comment]
Ok got it!

**Comment by mouzergrodno** [comment]
@leoluk 
Ow that's cool! But my problem I'm not so technically inclined, is it possible to get any step-by-step description if what I should do?? Thank you

**Comment by mouzergrodno** [comment]
@leoluk 
I can't create a token account...I don't see any option to do this on the link that you sent....is it possible to send me a more detailed tutorial how to do it?

**Comment by leoluk** [comment]
See https://github.com/solana-labs/oyster/issues/196#issuecomment-909535731 for instructions - do a "pretend" transfer with a small amount for the same token pair, but only complete the first transaction on the Solana side.

**Comment by mouzergrodno** [comment]
@leoluk
Should I do it where? On Wormhole V1? https://v1.wormholebridge.com - here?

**Comment by leoluk** [comment]
Yes

**Comment by mouzergrodno** [comment]
@leoluk
Hi! I have a problem - can't create a token account, because there's no WBTC in the new list of accepted tokens lol....but back then when I did it the list of tokens was much more expanded, and there was WBTC and ETH.....you could transfer ETH or WBTC to Solana, but couldn't do anything with these tokens over there, there was no liquidity for it, so you couldn't even swap it, so people just sent it back to Ethereum blockchain using the same bridge.....so now it's fixed and you can't even transfer these tokens using Wormhole bridge....so what should I do?

**Comment by leoluk** [comment]
Oh... I see. We'll need some custom code, I suppose - I'll get back to you!

**Comment by mouzergrodno** [comment]
Ok...thank you

**Comment by mouzergrodno** [comment]
@leoluk 
Hi! Any updates on my case?

**Comment by mouzergrodno** [comment]
@leoluk 
Hi! Can you give me some real ETA on the time when I can get my btc back? In January you said it's gonna take a few weeks,  now it's April and I still don't have it....thank you.

**Comment by mouzergrodno** [comment]
@leoluk 
Hi! Any updates???

**Comment by mouzergrodno** [comment]
HI! Can someone give me any updates on my case??? It's been two months and I don't have any feedback here

**Comment by ckeun** [comment]
Hi @mouzergrodno 👋 Okay, so i've been able to recreate your issue and was able to do this "fake" transfer from ETH->SOL in order to set up the associated token account. The main issue right now is that the website for the V1 bridge is no longer being hosted. But you can spin up your own local UI to do the transfer. I've written up instructions on how to spin up the UI & do the "fake" transfer if you wanted to take a look. 

[v1bridge_setup_wbtc2.pdf](https://github.com/solana-labs/oyster/files/8600725/v1bridge_setup_wbtc2.pdf)

**Comment by mouzergrodno** [comment]
@ckeun 
Hi! Thank you for your reply!
I'm kinda confused, cause I'm not a developer and English is not my first language...that's the only way to do it??? Is it possible if you do part 1, send me a link and I will do part 2 where I connect my wallets and do fake transaction???
Anyway, I will try to follow your instruction and do it but I'm sure I ll encounter some issues right away cause I never done something like that...I'm just a crypto user, but it sounds like a challenge and I like trying new staff...
So please let me know if that's the only option we have...thank you

**Comment by ckeun** [comment]
I don't know of a different way at the moment.... please give this a try 🙏

**Comment by mouzergrodno** [comment]
@ckeun Hi...I gave it a try....I'm doing it on Windows, so "sudo" commands didn't work ( you probably use Linux), but I checked some tutorials, so I installed yarn and lerna, but at 1c step I run "yarn bootstrap" and it was done, but the next one - "lerna exec yarn install --scope @oyster/common" didn't work - I got errors....sending you a screenshot...
Letting you know again that I'm not a developer or tester, I'm just a crypto user and I just want my money back...
![Screenshot (22)](https://user-images.githubusercontent.com/90993284/169166831-dd16da78-a671-4481-9621-20d177dc930b.png)

**Comment by mouzergrodno** [comment]
@ckeun 

./src/routes.tsx
Module not found: Can't resolve '@oyster/common' in 'C:\Users\WINDOWS\oyster\packages\bridge\src'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'
Error from chokidar (C:\node_modules): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'
Error from chokidar (C:\node_modules): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'
Error from chokidar (C:\node_modules): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'
Error from chokidar (C:\node_modules): Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'

Got further, but website isn't loading, showing this error - some path troubles in the script

**Comment by mouzergrodno** [comment]
@ckeun Hi! Any updates on my case??

**Comment by ckeun** [comment]
> 

Hi, what version of Windows are you running for this? I don't have a windows box setup currently but will try to recreate it on one.

**Comment by mouzergrodno** [comment]
I'm using Windows 10..
But it's not the problem, the actual path of the code is broken, there are many path troubles like common/oyster but actually folder oyster is located in another folder not common and lots examples like that....I was doing it with a friend who is a developer so he managed to adjust the commands for Windows, we got to localhost but it just didn't work...and he said that we did everything right it's just problem with a code

**Comment by ckeun** [comment]
Well, I got a hold of a windows 10 machine and was able to get the UI up & running. Obviously, it involved walking over some broken glass, but hopefully you'll be able to replicate it 🙏 . I installed WSL and i think that made my life a lot easier than trying to navigate without it. Here's the updated instructions. Hopefully I tripped over all of the gotchas... 😢 
[v1bridge_setup_wbtc_windows10.pdf](https://github.com/solana-labs/oyster/files/8980070/v1bridge_setup_wbtc_windows10.pdf)

**Comment by mouzergrodno** [comment]
@ckeun Thank you very much! I will do my best, maybe this week or next week, I will keep you posted!

**Comment by Ddotmoe** [comment]
Is there a way to do this with UST wormhole terra I have the same issue

**Comment by ckeun** [comment]
> Is there a way to do this with UST wormhole terra I have the same issue

yeah, if this v1-wrapped UST.. i'm guessing on solana? You would follow the same instructions (if you're on a windows machine) to spin up a UI and go to this website: http://localhost:3000/#/move?from=SOL&token=UST

**Comment by Ddotmoe** [comment]
I was told that my funds were sent to the wormhole address which is a permissionless contract and I can't get them back. Is there no one in control of the wormhole addresses?

**Comment by mouzergrodno** [comment]
Well. I tried to run the website, but unsuccessful. 
yarn bootstrap looks to be working
<img width="472" alt="yarn bootstrap looks to be working" src="https://user-images.githubusercontent.com/90993284/178628542-ea2e5843-dbe3-45e5-9d78-43ec5f7fd0ba.png">
yarn build works
<img width="476" alt="yarn build works" src="https://user-images.githubusercontent.com/90993284/178628596-9da29cfa-09c2-4132-8633-09001342ad9b.png">
I tried to run yarn start from bridge package as well as yarn start lending from main directory and I get this:
<img width="477" alt="yarn start" src="https://user-images.githubusercontent.com/90993284/178628689-be0a2e5e-6514-4722-bfee-b4861cc7c174.png">
<img width="475" alt="yarn start lending command after 30 sec" src="https://user-images.githubusercontent.com/90993284/178628702-df51eed6-fd6d-4423-8758-dcdf07234d6b.png">
<img width="476" alt="yarn start before error" src="https://user-images.githubusercontent.com/90993284/178628707-771d1b2b-1eca-4dc7-ac41-31e6fc5a0d90.png">
<img width="475" alt="compile error" src="https://user-images.githubusercontent.com/90993284/178628710-53d1e925-7fe5-423b-8ba4-d08803616d5c.png">
<img width="477" alt="yarn start lending command" src="https://user-images.githubusercontent.com/90993284/178628726-8e3f8548-3f25-4f86-b513-af8baad4a104.png">

When I run yarn start it launches website in localhost, but page is empty and there are error in console:
<img width="904" alt="Error in browser" src="https://user-images.githubusercontent.com/90993284/178628821-33204e96-8051-4973-8044-ec13ee456d69.png">

I also tried to delete the package and do it again and got the same exact results. When I run the website and I get white screen, after trying to refresh in powershell I get this error
<img width="499" alt="error when refresh in browser" src="https://user-images.githubusercontent.com/90993284/178629147-322221fe-cbae-4dd1-b6d0-c062c04b2c23.png">
:

**Comment by mouzergrodno** [comment]
@ckeun 
HI! Disregard my messages above, I finally made it!!!! Site loaded! Can't even say how happy I am! I did a failed transaction, that's the hash from solscan
https://solscan.io/tx/5oY9pJQFG8H6379ffHdonC1mdXA4etrUgVmcfXzG8amAneoBLrL5pf34Q365GxKggpMTzc7k144Gw1TMK9vbnNNJ
So let me do what to do next? You just gonna send my wbtc back to my eth address right?????

**Comment by ckeun** [comment]
> Making progress here!
> 
> ```
> 2022-02-18T11:03:43.998Z	INFO	root	submitting signed VAA to Solana	{"digest": "03c15a26508e55d9ac9d1ca4b2daa2e899411e53d812ba2e77a98c1570e5ae33", "vaa": {"Version":1,"GuardianSetIndex":1,"Signatures":[{"Index":0,"Signature":[134,212,43,233,185,67,89,92,254,210,201,201,60,215,213,62,188,73,162,215,68,93,89,15,48,95,46,110,201,30,29,170,52,125,207,150,169,179,77,118,81,236,4,240,127,120,46,41,179,27,254,77,75,34,200,112,36,183,243,64,195,4,195,93,1]},{"Index":2,"Signature":[156,113,31,51,54,140,248,230,213,49,200,95,58,101,123,108,37,137,249,148,228,141,22,92,9,208,208,172,248,203,137,146,0,233,195,93,118,92,174,238,2,174,6,188,250,114,21,175,169,136,132,107,185,43,218,254,238,246,66,222,22,164,232,104,0]},{"Index":3,"Signature":[101,238,187,65,159,193,124,157,229,241,203,169,208,226,201,155,208,237,140,187,140,137,83,221,221,88,71,179,196,25,53,103,77,153,120,183,73,156,188,137,30,205,52,198,209,184,118,94,88,56,29,236,69,1,165,189,110,94,252,72,214,41,132,176,0]},{"Index":4,"Signature":[36,254,99,172,146,187,203,187,200,155,172,195,45,189,177,4,162,37,135,134,62,40,213,169,122,243,146,74,57,61,75,145,73,116,110,213,215,48,5,133,145,124,73,130,59,226,239,110,45,84,13,119,250,19,55,50,230,100,255,176,35,15,172,178,0]},{"Index":5,"Signature":[28,62,148,123,175,97,15,41,87,132,129,88,159,208,246,124,98,101,206,62,13,238,157,107,154,35,107,34,173,84,108,36,115,88,55,91,86,58,11,104,120,214,20,254,209,0,146,171,107,231,123,5,163,28,133,191,80,74,108,212,117,27,78,110,1]},{"Index":6,"Signature":[209,25,159,124,173,60,112,23,25,148,210,71,239,189,135,176,60,210,62,165,182,115,44,98,200,122,118,95,76,19,215,110,45,68,20,116,51,39,28,158,73,114,154,40,51,103,197,33,101,204,253,94,65,39,177,210,30,69,187,182,103,77,152,209,0]},{"Index":8,"Signature":[133,112,113,87,57,70,28,198,233,32,66,234,97,55,60,167,64,115,107,178,240,194,63,202,111,130,138,99,6,97,101,63,82,205,35,143,167,218,42,105,49,122,27,175,221,147,25,120,36,224,134,226,248,178,84,90,222,5,18,14,14,247,95,255,0]},{"Index":10,"Signature":[182,175,150,83,72,118,192,219,60,193,126,171,142,100,153,74,191,87,95,12,75,17,187,244,234,24,245,166,212,29,56,222,98,70,154,38,227,16,31,36,27,4,236,74,74,183,116,99,219,75,221,118,76,171,53,227,67,106,98,16,36,87,74,63,1]},{"Index":12,"Signature":[48,211,131,43,44,60,229,146,128,236,140,230,118,46,255,80,217,72,241,149,95,208,109,44,142,195,62,216,240,200,51,118,122,128,145,183,55,14,157,194,128,30,118,176,204,157,149,109,207,153,167,192,48,223,11,201,69,131,31,230,42,186,109,48,0]},{"Index":13,"Signature":[208,179,61,209,91,109,48,111,45,107,12,252,234,49,34,121,173,208,55,171,188,55,212,117,70,111,186,139,197,43,218,170,103,206,231,30,125,39,120,92,175,201,194,142,221,65,171,127,34,192,221,188,27,157,20,125,181,173,60,18,87,222,239,20,0]},{"Index":15,"Signature":[90,41,192,136,207,245,5,219,17,204,44,154,215,158,66,118,158,222,67,242,60,195,33,144,76,168,244,183,165,27,236,114,104,136,159,182,6,186,120,180,174,77,207,173,50,45,9,203,42,117,133,70,30,200,102,196,86,159,49,3,227,212,221,13,0]},{"Index":16,"Signature":[190,193,49,94,78,54,106,18,40,172,166,236,182,218,183,119,192,252,209,42,70,69,159,174,100,206,27,133,137,86,16,120,59,26,161,107,56,216,186,198,247,13,75,194,4,229,118,33,201,95,149,111,120,169,89,192,38,11,252,134,210,75,152,96,1]},{"Index":18,"Signature":[218,75,174,10,200,191,120,96,250,14,253,137,56,71,58,20,178,172,83,45,234,186,58,26,36,135,9,216,207,137,42,255,19,126,153,139,168,185,243,175,12,219,115,43,45,145,69,195,130,2,232,116,168,171,76,54,122,142,142,45,38,229,197,199,1]}],"Timestamp":"2021-09-15T02:21:36Z","Payload":{"Nonce":159,"SourceChain":2,"TargetChain":1,"SourceAddress":"00000000000000000000000015f5c356656abba5a4b51f342b65a2b0ed36ab71","TargetAddress":"05369929e74ff10aca1c21635f2203e821d2adfa70616cc32e6cf38a30343720","Asset":{"Chain":2,"Address":"0000000000000000000000002260fac5e5542a773aa44fbcfedf7c193bc2c599","Decimals":8},"Amount":60000000}}, "bytes": "01000000010d0086d42be9b943595cfed2c9c93cd7d53ebc49a2d7445d590f305f2e6ec91e1daa347dcf96a9b34d7651ec04f07f782e29b31bfe4d4b22c87024b7f340c304c35d01029c711f33368cf8e6d531c85f3a657b6c2589f994e48d165c09d0d0acf8cb899200e9c35d765caeee02ae06bcfa7215afa988846bb92bdafeeef642de16a4e868000365eebb419fc17c9de5f1cba9d0e2c99bd0ed8cbb8c8953dddd5847b3c41935674d9978b7499cbc891ecd34c6d1b8765e58381dec4501a5bd6e5efc48d62984b0000424fe63ac92bbcbbbc89bacc32dbdb104a22587863e28d5a97af3924a393d4b9149746ed5d7300585917c49823be2ef6e2d540d77fa133732e664ffb0230facb200051c3e947baf610f29578481589fd0f67c6265ce3e0dee9d6b9a236b22ad546c247358375b563a0b6878d614fed10092ab6be77b05a31c85bf504a6cd4751b4e6e0106d1199f7cad3c70171994d247efbd87b03cd23ea5b6732c62c87a765f4c13d76e2d44147433271c9e49729a283367c52165ccfd5e4127b1d21e45bbb6674d98d100088570715739461cc6e92042ea61373ca740736bb2f0c23fca6f828a630661653f52cd238fa7da2a69317a1bafdd93197824e086e2f8b2545ade05120e0ef75fff000ab6af96534876c0db3cc17eab8e64994abf575f0c4b11bbf4ea18f5a6d41d38de62469a26e3101f241b04ec4a4ab77463db4bdd764cab35e3436a621024574a3f010c30d3832b2c3ce59280ec8ce6762eff50d948f1955fd06d2c8ec33ed8f0c833767a8091b7370e9dc2801e76b0cc9d956dcf99a7c030df0bc945831fe62aba6d30000dd0b33dd15b6d306f2d6b0cfcea312279add037abbc37d475466fba8bc52bdaaa67cee71e7d27785cafc9c28edd41ab7f22c0ddbc1b9d147db5ad3c1257deef14000f5a29c088cff505db11cc2c9ad79e42769ede43f23cc321904ca8f4b7a51bec7268889fb606ba78b4ae4dcfad322d09cb2a7585461ec866c4569f3103e3d4dd0d0010bec1315e4e366a1228aca6ecb6dab777c0fcd12a46459fae64ce1b85895610783b1aa16b38d8bac6f70d4bc204e57621c95f956f78a959c0260bfc86d24b98600112da4bae0ac8bf7860fa0efd8938473a14b2ac532deaba3a1a248709d8cf892aff137e998ba8b9f3af0cdb732b2d9145c38202e874a8ab4c367a8e8e2d26e5c5c701614158b0100000009f020100000000000000000000000015f5c356656abba5a4b51f342b65a2b0ed36ab7105369929e74ff10aca1c21635f2203e821d2adfa70616cc32e6cf38a30343720020000000000000000000000002260fac5e5542a773aa44fbcfedf7c193bc2c599080000000000000000000000000000000000000000000000000000000003938700"}
> ```
> 
> https://explorer.solana.com/tx/38D9sMtRArn8HW1jPUPN3FN7NN4jJzAijdTD7aoA65B7RgoY7tPkuw1gG53wAfY6b4Tuon21aWQKf5ecw4eUmVag
> 
> You'll have to create the token account for the transfer first, please refer to these instructions: [#196 (comment)](https://github.com/solana-labs/oyster/issues/196#issuecomment-909535731)
> 
> Once that is done, I'll retry the submission.

hey @leoluk, @mouzergrodno has created the ATA for the WBTC (V1). Can you try a resubmission for their transaction?

**Comment by ckeun** [comment]
Okay, @mouzergrodno. the transfer of 0.6 wBTC to your solana wallet should be completed. 
https://explorer.solana.com/tx/3tiCW21A5jBsJkxaj1fKnuwvNqB2iUbpSmXMcNAo6S7AMPtt2TyBaLnBvA19QhT2psuxXji7RQUcXM7iCVDBGdmD

Now in order to get the WBTC back to ethereum, use the UI that you've spun up & switch the url to: http://localhost:3000/#/move?from=SOL&token=WBTC

Do the same thing you did before. Connect both wallets, WBTC should already be chosen, but if it's not, try the token pull down. First do a test transfer of a small amount. And when that's successful, transfer the rest. 👍

**Comment by mouzergrodno** [comment]
@leoluk 
@ckeun 
Everything worked! Got my money back!!!! Thank you very much both for your help! Really appreciate it!!!!

**Comment by opoplol7** [comment]
> Well, I got a hold of a windows 10 machine and was able to get the UI up & running. Obviously, it involved walking over some broken glass, but hopefully you'll be able to replicate it 🙏 . I installed WSL and i think that made my life a lot easier than trying to navigate without it. Here's the updated instructions. Hopefully I tripped over all of the gotchas... 😢 [v1bridge_setup_wbtc_windows10.pdf](https://github.com/solana-labs/oyster/files/8980070/v1bridge_setup_wbtc_windows10.pdf)

hello,
I have tied the step in you doc but i am stuck with the web site hanging as you told in it don't understand what to do next :/
thanks for any help

**Comment by ckeun** [comment]
hey, can you paste your public wallet address so i can look it up on solscan? & are you running it on a mac/linux/windows?

**Comment by opoplol7** [comment]
> hey, can you paste your public wallet address so i can look it up on solscan? & are you running it on a mac/linux/windows? what are you trying to bridge back?

thanks for answering me,
CfAyDnXYhnGxVkHCfippwSsFdfC17WWcjJZumer3dony this is my address on sol but i don't think you will see anything on chain i havn't done an tx since the website crash when i click on connect button even if every thing seems to work before i do that.

**Comment by opoplol7** [comment]
@ckeun do you see the issue ?

**Comment by ckeun** [comment]
I've never seen that issue before. 😞  If you haven't already, can you try rebuilding & relaunching the website? Also just looking at the screenshot... it says "Click X or hit esc to dismiss the message" .. when you do that.. what happens?

**Comment by opoplol7** [comment]
> I've never seen that issue before. 😞 If you haven't already, can you try rebuilding & relaunching the website? Also just looking at the screenshot... it says "Click X or hit esc to dismiss the message" .. when you do that.. what happens?

LOL i didn't read the end of the message, and it seems to work with esc.. ty 
SO now i have clicked transfert but there is no pop up from wallet.
![image](https://user-images.githubusercontent.com/99958331/204130191-569213d4-65b0-46be-817c-ca6b10711627.png)

**Comment by ckeun** [comment]
oh interesting. yeah i'd expect the dai balance to show up on the solana side 🤔 . i wonder if this is an rpc issue with the phantom wallet & it can't get the token balances? i'll try to replicate it on my side. super weird.

**Comment by opoplol7** [comment]
> I've never seen that issue before. 😞 If you haven't already, can you try rebuilding & relaunching the website? Also just looking at the screenshot... it says "Click X or hit esc to dismiss the message" .. when you do that.. what happens
on localhost my phatom do this maybe there is a pb i don't know

![image](https://user-images.githubusercontent.com/99958331/204161663-a1239791-bdd9-4aeb-956e-f4a5215b0374.png)

**Comment by opoplol7** [comment]
> yeah i'm getting the same error too. i'll try to figure it out & get back to you.

thanks!

**Comment by opoplol7** [comment]
> yeah i'm getting the same error too. i'll try to figure it out & get back to you.
Hey @ckeun did you find a solution yet?

**Comment by ckeun** [comment]
No, I couldn't figure out how to get it to work :( It looks like the rpc that powered the backend got disconnected. I tried to  replace it with a diff rpc but then couldn't complete transfers from solana to ethereum. Maybe something changed with the dependencies or something in the past few months? The code base for the v1 bridge hasn't been managed/supported for a while now... so i'm not sure what else can be done. 😢

**Comment by opoplol7** [comment]
> No, I couldn't figure out how to get it to work :( It looks like the rpc that powered the backend got disconnected. I tried to replace it with a diff rpc but then couldn't complete transfers from solana to ethereum. Maybe something changed with the dependencies or something in the past few months? The code base for the v1 bridge hasn't been managed/supported for a while now... so i'm not sure what else can be done. 😢

😢So you think my found is lost?

**Comment by opoplol7** [comment]
> is

Is it you who were in charge of wormholev1 bridge @ckeun ?

**Comment by ckeun** [comment]
> > is
> 
> Is it you who were in charge of wormholev1 bridge @ckeun ?

no i was just initially tasked with helping folks migrate from v1 to v2/portal in the early part of 2022. there hasn't been active support for this for several months now. & i'm basically doing this in my free time 💔 sorry @opoplol7 😢

**Comment by opoplol7** [comment]
> > > is
> > 
> > 
> > Is it you who were in charge of wormholev1 bridge @ckeun ?
> 
> no i was just initially tasked with helping folks migrate from v1 to v2/portal in the early part of 2022. there hasn't been active support for this for several months now. & i'm basically doing this in my free time 💔 sorry @opoplol7 😢

Ok thank for your help @ckeun if somehow you found a way tell me I will give something like 50$
